### PR TITLE
Update bok choy tests to have unique emails.

### PR DESCRIPTION
**Pull request by cahrens**
@efischer19 and @dianakhuang Please review. Note that I need the email in order to log back in as an existing user-- that's why I was using a specific email in the past. Now I am returning the generated unique email from the auto auth code.

All tests passed with these changes on my sandbox.

### CODE REVIEW

File: test/acceptance/auto_auth.py [hunk 1]
```diff
@@ -82,11 +82,24 @@ def get_user_id(self):
...
+        """
+        Finds and returns the email address of the current user.
+        """
+        username_and_email = self._get_username_and_email()
+        return username_and_email.split(' ')[1]
+
+    def _get_username_and_email(self):
+        """
+        Finds and returns a string containing username and email address of the current user.
         """
```
Comment (dianakhuang) [inline test/acceptance/auto_auth.py]: I think it makes sense to return the username and email as a tuple instead of as a string.

Comment (cahrens) [inline test/acceptance/auto_auth.py]: It was just an internal helper method, but I will change it.

**Comment by dianakhuang** [comment]
Other than that one comment, :+1:

**Comment by efischer19** [comment]
Looks great! :+1:

**Pull request merged**